docs/rst/user_guide/atmosphere/intro.rst

```diff
@@ -0,0 +1,209 @@
+.. _sec-atmosphere-intro:
+
+Introduction
+============
+
+Assumptions
+-----------
+
+The air is assumed to behave as an
+`ideal gas <https://en.wikipedia.org/wiki/Ideal_gas>`_,
+i.e. a collection of point particles without interactions between one another.
+It is in thermodynamic equilibrium, it is not chemically reacting and it obeys
+the ideal gas state equation:
+
+.. math::
+
+   p = n k T
+
+where:
+
+* :math:`p` stands for the pressure :math:`[ML^{-1}T^{-2}]`,
+* :math:`n` stands for the number density :math:`[L^{-3}]`,
+* :math:`k` is the
+  `Bolzmann constant <https://en.wikipedia.org/wiki/Boltzmann_constant>`_
+  :math:`[ML^{2}T^{-2}\Theta^{-1}]` and
+* :math:`T` stands for the temperature :math:`[\Theta]`.
+
+Air can usually be treated as an ideal gas within reasonable tolerance over a
+wide parameter range around standard temperature and pressure (273 K, 100 kPa)
+and the approximation generally gets better with lower pressure and higher
+temperature.
+
+Atmosphere modelling
+--------------------
+
+Eradiate represents the atmosphere as a **shape** that encompasses a
+**participating medium** (the air).
+
+The shape is a cuboid characterised by width and height values that correspond
+to the width and height, respectively, of the atmosphere object.
+
+.. image:: fig/atmosphere-cuboid_shape.png
+   :align: center
+   :scale: 50
+
+This shape delimits the region of space that is occupied by the participating
+medium.
+
+The participating medium is characterised by a phase function type and two 3D
+arrays of albedo and extinction coefficient values that describe how
+radiative properties vary in space.
+
+.. image:: fig/atmosphere-participating_medium.png
+   :align: center
+   :scale: 50
+
+Each value in these 3D array corresponds to one cell of a spatial mesh that
+discretises the participating medium into an arrangement of adjacent 3D cells
+wherein the radiative properties are uniform.
```

Comment:
That's technically not true: the volume texture object can be configured to use nearest neighbour interpolation, but also linear. However, if, for consistency with the assumptions made to build the thermophysical profile, we should restrict to the nearest neighbour case, it's fine.

Reply:
Ah, I forgot about that. When building the thermophysical profiles, it is indeed assumed that the atmosphere cells are homogeneous.